ui/raidboss/common_replacement.js

```diff
@@ -91,6 +91,20 @@ let commonReplacement = {
     cn: '--西北--',
     ko: '--북서--',
   },
+  '--orbs--': {
+    de: '--kugeln--',
+    fr: '--orbes--',
+    ja: '--オーブ--',
+    cn: '--球--',
+    ko: '--오브--',
+  },
+  '--regulator check--': {
```

Comment:
Wouldn't it be better here? Yes we may not will see it anymore, but I find it simpler to have the translations with the dashes here, especially since these are loading times and indications where there is no DPS. These are translations that will never move.
Tell me what you think.

Reply:
No, my feeling is that "common replacement" is for replacements that are common.  I think replacements that are only in one fight do not meet the criteria for "common" to me.  I am glad that file is useful, but I think if there is too much in there, than it becomes less useful and confusing.  I think a translation needs to be in ~3 different fights before I would consider it "common".

Reply:
Ok ^^.